proposals/typography-components.md

```diff
@@ -0,0 +1,64 @@
+# Typography package
+
+This proposal is to create a `baseui/typography` package containing the typography components. These components codify the [BaseUI typography standards](https://www.figma.com/file/L7MSVAit4clxkfGE22NMX86Q/%E2%9D%96-Base-UI?node-id=19590%3A824) for H1 .. H6, Display, Caption, CaptionLabel, Label1, Label2, P1, P2,
```

Comment:
removed the figma link since it's not publicly available

```suggestion
This proposal is to create a `baseui/typography` package containing the typography components. These components codify the BaseUI typography standards for H1 .. H6, Display, Caption, CaptionLabel, Label1, Label2, P1, P2,
```